src/algorithms.rs

```diff
@@ -51,19 +88,155 @@ impl Algorithm for ChaosGame {
     }
 
     fn save(&mut self, fname: &str) {
-        let mut writer = Cesium3DTilesWriter::new(10000000.0);//CSVWriter::new();
+        let mut writer = Cesium3DTilesWriter::new(10000000.0);
```

Comment:
Add a TODO about selecting between CSV and 3D tiles.

Also, name this magic number to indicate that we're scaling to the size of the earth.